<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path to mail dir:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_path_mailbox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_alwayswebview">
     <property name="text">
      <string>Always use webview to display messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_automaticallywebview">
     <property name="text">
      <string>Use webview automatically, when needed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_checknew">
     <property name="title">
      <string>Check for new mail automatically</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>After</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sp_seconds_tocheck"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_trayicon">
     <property name="text">
      <string>Display icon in tray</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_notify">
     <property name="text">
      <string>Display notifications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_usexdgbrowser">
     <property name="text">
      <string>Use external broweser (through xdg)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
